src/scipp/logging.py

```diff
@@ -140,8 +140,9 @@ def display_logs() -> None:
             'Cannot display log widgets because no widget handler is installed.')
 
     from IPython.display import display
+    from ipywidgets import VBox
     inject_style()
-    display(widget)
+    display(VBox([widget]).add_class('sc-log-wrap'))
```

Comment:
Was it not possible to add the class directly to the widget instead of making an extra `VBox`?

Reply:
Ah sorry, these changes were from the other PR